internal/lsp/server.go

```diff
@@ -1096,6 +1113,32 @@ func (l *LanguageServer) StartWebServer(ctx context.Context) {
 	l.webServer.Start(ctx)
 }
 
+func (l *LanguageServer) determineVersionForFile(fileURI string) ast.RegoVersion {
+	var versionedDirs []string
+
+	// if we have no information, then we can return the default
+	if l.loadedConfigAllRegoVersions == nil || l.loadedConfigAllRegoVersions.Len() == 0 {
```

Comment:
Perhaps we should have Len() always return 0 when its parent is nil? It would make the check here a little cleaner.